src/components/Roadmap/index.tsx

```diff
@@ -14,8 +14,8 @@ const techCategories: TechCategory[] = [
     title: "Frontend",
     technologies: [
       { id: 1, name: "HTML", link: "/roadmap/html" },
-      { id: 2, name: "CSS", link: "#" },
-      { id: 3, name: "JavaScript", link: "#" },
+      { id: 2, name: "CSS", link: "" },
+      { id: 3, name: "JavaScript", link: "../database/Roadmap/Js" },
```

Comment:
It is not working Please tell changes to do

Reply:
1. Navigate to the `pages/roadmap` directory and create a new file named `javascript/index.tsx`.
2. Import the content from `database/Roadmap/Js` into this new file.
3. Write the code for the new JavaScript roadmap page in `javascript/index.tsx`.
4. Update the import path in `src/components/Roadmap/index.tsx` by replacing `../database/Roadmap/Js` with `/roadmap/javascript`.